pkgs/development/libraries/libproxy/default.nix

```diff
@@ -1,82 +1,77 @@
 { lib
-, stdenv
+, bash
+, curl
+, duktape
+, enableDuktape ? lib.meta.availableOn stdenv.hostPlatform duktape
 , fetchFromGitHub
 , fetchpatch
-, pkg-config
-, cmake
-, zlib
-, dbus
-, networkmanager
-, enableJavaScript ? stdenv.isDarwin || lib.meta.availableOn stdenv.hostPlatform duktape
-, duktape
-, pcre
-, gsettings-desktop-schemas
+, gi-docgen
 , glib
-, makeWrapper
-, python3
-, SystemConfiguration
+, gobject-introspection
+, gsettings-desktop-schemas
+, meson
+, ninja
+, pkg-config
+, stdenv
+, vala
 , CoreFoundation
 , JavaScriptCore
+, SystemConfiguration
 }:
 
 stdenv.mkDerivation rec {
   pname = "libproxy";
-  version = "0.4.18";
+  version = "0.5.3";
+
+  outputs = [ "out" "dev" ];
 
   src = fetchFromGitHub {
     owner = "libproxy";
     repo = "libproxy";
     rev = version;
-    hash = "sha256-pqj1LwRdOK2CUu3hYIsogQIXxWzShDuKEbDTbtWkgnQ=";
+    hash = "sha256-qdYB6HJkgboS8kkTvTqLy6Z3JYY5SOJsRl6nZM0iuvw=";
   };
 
-  patches = lib.optionals stdenv.isDarwin [
-    # https://github.com/libproxy/libproxy/pull/189
+  patches = [
     (fetchpatch {
-      url = "https://github.com/libproxy/libproxy/commit/4331b9db427ce2c25ff5eeb597bec4bc35ed1a0b.patch";
-      sha256 = "sha256-uTh3rYVvEke1iWVHsT3Zj2H1F+gyLrffcmyt0JEKaCA=";
+      url = "https://github.com/libproxy/libproxy/pull/261/commits/fe64d1f56b7695bbc8ccb31a34546bc289a4c44d.patch";
+      hash = "sha256-X13rXtmWvssdxy795Ag+2op1vUpEqt6+aicR64zKQnQ=";
     })
   ];
 
-  outputs = [ "out" "dev" "py3" ];
+  postPatch = ''
+    # prevent meson from installing git hooks...
+    # https://github.com/libproxy/libproxy/issues/262
+    echo -e "#!${lib.getExe bash}\nexit 0" > data/install-git-hook.sh
+    chmod +x data/install-git-hook.sh
+  '';
 
   nativeBuildInputs = [
+    gi-docgen
```

Comment:
Use `devdoc` output? And move the docs there manually until https://github.com/NixOS/nixpkgs/pull/187042 is merged?